mne/viz/topo.py

```diff
@@ -61,14 +62,32 @@ def iter_topography(info, layout=None, on_pick=None, fig=None,
 
     Returns
     -------
-    A generator that can be unpacked into
+    A generator that can be unpacked into:
+
+        ax : matplotlib.axis.Axis
+            The current axis of the topo plot.
+        ch_dx : int
+            The related channel index.
 
-    ax : matplotlib.axis.Axis
-        The current axis of the topo plot.
-    ch_dx : int
-        The related channel index.
     """
-    import matplotlib.pyplot as plt
+    return _iter_topography(info, layout, on_pick, fig, fig_facecolor,
+                            axis_facecolor, axis_spinecolor, layout_scale)
+
+
+def _iter_topography(info, layout, on_pick, fig,
+                     fig_facecolor='k', axis_facecolor='k',
+                     axis_spinecolor='k', layout_scale=None,
+                     unified=False):
```

Comment:
possibly, but then the order of the arguments wouldn't match with `iter_topography` above, which would I think would hurt readability more than saving a line